exe/kubernetes-deploy

```diff
@@ -82,7 +82,6 @@ begin
 
     runner.run!(
       verify_result: !skip_wait,
-      allow_protected_ns: allow_protected_ns,
```

Comment:
I think we need to maintain backwards compatibility of the exe commands until we delete them.

Reply:
Makes sense. Updating...